test/memspaces/memspace_fixtures.hpp

```diff
@@ -29,7 +29,7 @@ struct numaNodesTest : ::umf_test::test {
         ::umf_test::test::SetUp();
 
         if (numa_available() == -1 || numa_all_nodes_ptr == nullptr) {
```

Comment:
Please, create a common function for this check in this struct and call it in inheriting structs.

Reply:
What do you want exactly?

Something like this will not work: 
``` c
void checknuma(){
    if (numa_available() == -1 || numa_all_nodes_ptr == nullptr) {
        GTEST_SKIP() << "XXX"
    }
}

```
Unless you will use it in this way:
``` c
  checknuma()
  if (IsSkipped()) return;
```
What is probably pointless

Reply:
OK, leave it then.